### DISCUSSION TITLE
BE URL not working as expected

### DISCUSSION THREAD

**Discussion by lade6501** [Luson045/medi-connect#394] [General]
Hi @Luson045 / @Trevisx07 , I discovered something not sure if it's a bug or some deployment issue.
The `https://medi-connect-f671.onrender.com/auth/register` endpoint deployed is not working as expected every time I try to register it give `Pincode is required` error.
But if switch to localhost url `http://localhost:3001/auth/register` it does not throw any error and I'm successfully able to `register`. Could you check from your side if it working or only I'm facing the issue.

Demo 1: With deployed BE url

https://github.com/user-attachments/assets/9c2c73c6-d774-4bb1-8fa8-dcf12c19b64a

Demo 2: With localhost BE url

https://github.com/user-attachments/assets/e61b6dc2-cc03-422c-afee-c23b31237367

**Comment by Luson045** [comment]
@lade6501  i think I haven't allowed auto deployment in the backend, I'll update it then see if it is working.

**Comment by Luson045** [comment]
@lade6501 can you check if we are sending pincode as it is to the backend?

**Comment by lade6501** [comment]
Yes we are sending  pincode to BE in payload insde address object  as `postalCode`. I tried console logging as well it's printing out the provided pincode from `req.body.address`

**Comment by Luson045** [comment]
![Screenshot_2024-10-12-14-43-35-13_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/996cf77c-5c2f-4883-b415-a03cfd87bca8)

It's working now, you can check.

**Reply by lade6501** [reply]
How much time it's taking to register, it more than 5 min not received any response from BE.

**Reply by Luson045** [reply]
@lade6501  sometimes it will take time, as render free instance spins down due the deactivity for certain time
